source/localizable/routing/defining-your-routes.md

```diff
@@ -29,6 +29,16 @@ Router.map(function() {
 });
 ```
 
+If your route name has multiple works then your route should follow convention and be hyphenated. Routes with underscores in them will not be rendered.
```

Comment:
@rwjblue should we be mentioning ember-cli here and using it as an explanation? I mean, if the route uses an underscore is ember-cli the reason a route will fail to render?